pyspecProcScripts/load_data.py

```diff
@@ -462,6 +500,26 @@ def proc_nutation_chunked(s, fl=None):
     return s
 
 
+def proc_nutation_v2(s, fl=None):
+    logging.info("loading pre-processing for nutation")
+    s.set_units("indirect", "s")
+    s.ft(["ph1"], unitary=True)
```

Comment:
you have to decide, in general, if this is going to be unitary or not, and be consistent across all postproc